src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -1185,31 +1179,30 @@ Fonte:
   - *Hyper-v*: #link("https://learn.microsoft.com/it-it/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v") (22-02-2024).
 
 #pagebreak()
+*/
 == Requisiti prestazionali
 
+/*
+HO TOLTO QUESTA SEZIONE PERCHè NON NE CAPIVO IL SENSO QUI
+
 Si nota che entrambe le tecnologie #glossary[Clickhouse] e #glossary[Grafana] integrano al loro interno varie ottimizzazioni per garantire prestazioni efficienti.
 Ad esempio #glossary[Clickhouse] utilizza algoritmi di compressione avanzati per ridurre lo spazio di archiviazione necessario e migliorare le prestazioni di accesso ai dati, inoltre ha un archittetura distribuita consentendo una distribuzione del carico e una scalabilità orizzontale per gestire grandi volumi di dati. Mentre #glossary[Grafana] usa la cache dei dati per ridurre il carico di lavoro sui database e migliorare la velocità di risposta delle query; cerca di ottimizzare le query per ridurre il numero di richieste e utilizza un sistema di compressione dati per ridurre lo spazio di archiviazione.
 
 Per garantire tempi di esecuzione ancora più rapidi, si cerca di usare, ove possibile, le #glossary[materialized views] di #glossary[Clickhouse].
+*/
 
 #let requisiti_prestazioni = (
   (
-  "Obbligatorio",[Il #glossary("sistema") deve gestire un carico di #glossary("dati in entrata") tra i 50 e i 100 dati al secondo per un sistema con processore multicore con almeno 2.5GHz di clock, 8 GB di RAM.],"Verbale esterno"
+  "Obbligatorio",[Il #glossary("sistema") deve gestire un carico di #glossary("dati in entrata") superiore ai 50 dati al secondo per un sistema con processore multicore con almeno 2.5GHz di clock, 8 GB di RAM.],"Verbale esterno"
```

Comment:
cosa dovrebbe succedere se i dati in ingresso sono meno di 50? 
rimango dell'idea che sia più utile stabilire un limite superiore affinchè le prestazioni non calino notevolmente, piuttosto che uno inferiore; per ora lascerei 100 ma con l'idea di rivisitarlo a prodotto "finito" per fare una stima/stress test

Reply:
il limite superiore non credo abbia senso, tu garantisce le performance minime di quelle massime cosa ti interessa, 
con questo requisiti noi stiamo dicendo il sistema deve almeno gestirne 50, se non lo fa il requisiti non è soddisfatto

Reply:
ma come è possibile lol